README.markdown

```diff
@@ -20,9 +20,9 @@ Jekyll is a simple, blog-aware, static site generator perfect for personal, proj
 
 Jekyll does what you tell it to do — no more, no less. It doesn't try to outsmart users by making bold assumptions, nor does it burden them with needless complexity and configuration. Put simply, Jekyll gets out of your way and allows you to concentrate on what truly matters: your content.
 
-## Having trouble with OS X El Capitan?
+## Having trouble with macOS 10.11 El Capitan?
```

Comment:
El Cap is still referred to as **OS X** on [Apple’s website](http://www.apple.com/osx/), I hesitate to deviate from that.